src/briefcase/macos.py

```diff
@@ -12,6 +12,37 @@ class macos(app):
     description = "Create a macOS app to wrap this project"
 
     def finalize_options(self):
+        # Before we start, exit if xcode is not installed
+        print()
+        print(" * Looking for Xcode...")
+        if subprocess.Popen(
+            [
+                '/usr/bin/xcodebuild', '-version'
+            ]
+        ).wait() == 1:
+            # raise error
+            print("Xcode is required.  Couldn't find Xcode.")
```

Comment:
Suggested message content: 

Briefcase requires Xcode, but it does not appear to be installed.
Please install the latest version of Xcode using the Mac App Store:

     https://itunes.apple.com/au/app/xcode/id497799835

Reply:
Okay, changed, but I replaced `au` with `us`